devkit/samples/attachments/attachments.feature.ts

```diff
@@ -1,10 +1,6 @@
 import { Before, When } from '@cucumber/fake-cucumber'
-import { ReadableStreamBuffer } from 'stream-buffers'
 import fs from 'fs'
 
-// Cucumber-JVM needs to use a Before hook in order to create attachments
```

Comment:
This looks like it will cause difficulties for the JVM integration. Do we have a way to work around it on the JVM side @mpkorstanje?

Reply:
If it's easier to re-add this in I can do. This was one I wasn't 100% sure on. Because at the time we didn't have a direct consumer for jvm. But if needs be I'll just revert this bit

Reply:
Yeah let's re-add this for now and raise a new issue to resolve it properly.